src/test/java/org/openrewrite/java/testing/junit5/AddParameterizedTestAnnotationTest.java

```diff
@@ -0,0 +1,50 @@
+package org.openrewrite.java.testing.junit5;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class AddParameterizedTestAnnotationTest implements RewriteTest {
+    @Test
```

Comment:
Feel free to link these back to specific issues, such as
```suggestion
    @Test
    @Issue("https://github.com/openrewrite/rewrite-testing-frameworks/issues/314")
```